src/Sulu/Bundle/AdminBundle/Resources/js/components/Modal/tests/Modal.test.js

```diff
@@ -0,0 +1,32 @@
+/* eslint-disable flowtype/require-valid-file-annotation */
+import {mount, shallow} from 'enzyme';
+import Modal from '../Modal';
+import React from 'react';
+import pretty from 'pretty';
+
+afterEach(() => document.body.innerHTML = '');
+
+test('The component should render in body when open', () => {
+    const body = document.body;
+    const view = mount(<Modal isOpen={true}><p>Modal content</p></Modal>).render();
+    expect(view.html()).toBe(null);
+    expect(pretty(body.innerHTML)).toMatchSnapshot();
```

Comment:
As mentioned, the only thing in the body is the stuff of the portal and that's exactly the things a snapshot should be taken of.